src/formio/templates/file.ejs

```diff
@@ -6,10 +6,10 @@
         {% if (!ctx.disabled) { %}
           <div class="col-md-1"></div>
         {% } %}
-        <div class="col-md-{% if (ctx.self.hasTypes) { %}7{% } else { %}9{% } %}"><strong>{{ctx.t('File Name')}}</strong></div>
-        <div class="col-md-2"><strong>{{ctx.t('Size')}}</strong></div>
+        <div id="{{ctx.instance.id}}-{{ctx.component.key}}__file-list-header--file-name" class="col-md-{% if (ctx.self.hasTypes) { %}7{% } else { %}9{% } %}"><b>{{ctx.t('File Name')}}</b></div>
```

Comment:
`strong` is typically used for its semantic meaning, and that has pretty much made `b` obsolete since that's only meant for visual presentation, which you can (and should) do with CSS.

So if the `strong` tag is an a11y problem, then we should remove it and apply the relevant/proper CSS instead of replacing it with `b` through a class name.

Reply:
Ah shit, i thought i removed those...

Yep, will replace them